url-shortener-db/url_shortener_db/base_62_convertion.py

```diff
@@ -0,0 +1,25 @@
+class Base62Convertion:
+    def __init__(self):
+        self.BASE62 = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
```

Comment:
I did smth like 
``` Python
class Base62Convertion:
    BASE62 = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
    BASE62_len = len(BASE62)
```
I wanted it to be in class because it is used only in this class.
Can we leave it like this?

Reply:
Sure